hardhat.gasUsage.config.ts

```diff
@@ -0,0 +1,19 @@
+import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat'
+import 'hardhat-gas-reporter'
+import deepmerge from 'deepmerge'
+import config from './hardhat.config'
+
+const overrides = {
+  mocha: {
+    invert: false,
+    timeout: 300_000,
+  },
+  gasReporter: {
+    enabled: true,
+    currency: 'USD',
+    coinmarketcap: process.env.CMC_KEY,
```

Comment:
Is this optional?

Reply:
yeah, optional (doesn't seem to work actually, I think the gas usage reporter is a little stale)